extensions/dacpac/package.json

```diff
@@ -37,6 +37,16 @@
 					"command": "dacFx.start",
 					"when": "connectionProvider == MSSQL && nodeType && nodeType == Database",
 					"group": "export"
+				},
+				{
+					"command": "dacFx.start",
+					"when": "connectionProvider == MSSQL && nodeType && nodeType == Server",
+					"group": "export"
+				},
+				{
+					"command": "dacFx.start",
+					"when": "connectionProvider == MSSQL && nodeType && nodeType == Folder && nodeLabel == 'Databases'",
```

Comment:
Can 'Databases' be localized, making this eval to false?

Reply:
Good point, It seems there needs to be a new property like nodeName which is language independent.

Reply:
Can the nodetype be extended to include DatabasesFolder rather than common Folder or that will be too wide impacting?

Reply:
@udeeshagautam I would not recommend that, the scenario when you want to add a menu item for example 'refresh' for all the folders, it would be very hard.